code-security/admin_guide/scan-monitor/secrets-scanning.adoc

```diff
@@ -9,25 +9,45 @@ For identifying secrets, Prisma Cloud provides default policies that use domain-
 image::scan-results-secrets-ide.png[width=800]
 
 
+=== Validate Secrets
```

Comment:
@anaghapamidi this is a task that is not written like a task.

Reply:
to support the release, I am merging this, the content needs work so please address it next week.
cc: @tsmithv11